app/main/NameTag.tsx

```diff
@@ -40,7 +40,22 @@ function NameTag({
   const debouncedEffect = debounce(() => {
     setNameTagStatus(showNametag);
     setCurrentNameTag(inputValues);
-    const imageData = drawNametag(showNametag, inputValues, selectedWaveHand, waveHands);
+
+    const nametag: NameTagBadge = {
+        visible: showNametag,
+        fullName: inputValues[0],
+        preferredName: inputValues[1],
+        pronouns: inputValues[2],
+        disclosure: inputValues[3],
+    };
+
+    const handWave: HandWaveBadge = {
+        visible: selectedWaveHand !== null ? true : false,
+        waveText: selectedWaveHand !== null ? waveHands[selectedWaveHand] : null,
+    };
```

Comment:
I find it a little clearer to not repeat the `selectedWaveHand !== null` check twice.  Maybe something like:
```suggestion
    const handWave: HandWaveBadge =
       selectedWaveHand !== null ?
           {visible: true, waveText: waveHands[selectedWaveHand]} :
           {visible: false};
```

If you agree, let's make the same change in page.tsx